### PR TITLE
Dropdown fixes

### PR DESCRIPTION
**Pull request by brboi**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/webclient/user_menu/user_menu.scss [hunk 1]
```diff
@@ -8,9 +8,9 @@
     transform: translateY(-2px);
   }
 
-  .o_dropdown_menu {
-    // FIXME weird hack to keep menu from taking all viewport width
-    min-width: 100%;
-    position: sticky;
+  .o-popper-position {
```
Comment (brboi) [inline addons/web/static/src/webclient/user_menu/user_menu.scss:11]: .o_drodown_menu

File: addons/web/static/src/webclient/user_menu/user_menu.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="web.UserMenu" owl="1">
-        <Dropdown class="o_user_menu" t-on-dropdown-item-selected.stop="onDropdownItemSelected">
+        <Dropdown class="o_user_menu" position="'bottom-start'" tag="'li'" t-on-dropdown-item-selected.stop="onDropdownItemSelected">
```
Comment (brboi) [inline addons/web/static/src/webclient/user_menu/user_menu.xml:5]: remove `position`

File: addons/web/static/src/core/dropdown/dropdown_navigation_hook.js [hunk 3]
```diff
@@ -208,7 +205,11 @@ export function useDropdownNavigation() {
         enter: () => {
             const menuElement = getActiveMenuElement();
             if (menuElement) {
-                menuElement.el.click();
+                if (menuElement.isSubDropdown) {
+                    menuElement.openSubDropdown(true);
+                } else {
+                    menuElement.el.click();
+                }
```
Comment (aab-odoo) [inline addons/web/static/src/core/dropdown/dropdown_navigation_hook.js:212]: could/should this be tested?

Comment (brboi) [reply addons/web/static/src/core/dropdown/dropdown_navigation_hook.js:212]: off course.
done in dropdown_tests.js

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**